contrib/pg_stat_statements/expected/test_pg_stat_statements.out

```diff
@@ -0,0 +1,131 @@
+-- case it is planned by GPORCA. So disable GPORCA during tests.
+SET optimizer='off';
+SELECT pg_stat_statements_reset();
+ pg_stat_statements_reset 
+--------------------------
+ 
+(1 row)
+
+SELECT GROUPING (a) FROM t GROUP BY ROLLUP(a, b);
+ grouping 
+----------
+(0 rows)
+
+-- launch not equivalent query
+SELECT GROUPING (b) FROM t GROUP BY ROLLUP(a, b);
+ grouping 
+----------
+(0 rows)
+
+-- check that 2 queries have separate entries
+SELECT query, calls FROM pg_stat_statements ORDER BY query;
+                       query                       | calls 
+---------------------------------------------------+-------
+ SELECT GROUPING (a) FROM t GROUP BY ROLLUP(a, b); |     1
+ SELECT GROUPING (b) FROM t GROUP BY ROLLUP(a, b); |     1
+ SELECT pg_stat_statements_reset();                |     1
+(3 rows)
+
+-- check that different grouping options result in separate entries
+SELECT COUNT (*) FROM t GROUP BY ROLLUP(a, b);
+ count 
+-------
+(0 rows)
+
+SELECT COUNT (*) FROM t GROUP BY CUBE(a, b);
+ count 
+-------
+(0 rows)
+
+SELECT COUNT (*) FROM t GROUP BY GROUPING SETS(a, b);
+ count 
+-------
+(0 rows)
+
+SELECT COUNT (*) FROM t GROUP BY GROUPING SETS((a), (a, b));
+ count 
+-------
+(0 rows)
+
+SELECT COUNT (*) FROM t GROUP BY a, b;
+ count 
+-------
+(0 rows)
+
+SELECT query, calls FROM pg_stat_statements ORDER BY query;
+                            query                             | calls 
+--------------------------------------------------------------+-------
+ SELECT COUNT (*) FROM t GROUP BY a, b;                       |     1
+ SELECT COUNT (*) FROM t GROUP BY CUBE(a, b);                 |     1
+ SELECT COUNT (*) FROM t GROUP BY GROUPING SETS((a), (a, b)); |     1
+ SELECT COUNT (*) FROM t GROUP BY GROUPING SETS(a, b);        |     1
+ SELECT COUNT (*) FROM t GROUP BY ROLLUP(a, b);               |     1
+ SELECT GROUPING (a) FROM t GROUP BY ROLLUP(a, b);            |     1
+ SELECT GROUPING (b) FROM t GROUP BY ROLLUP(a, b);            |     1
+ SELECT pg_stat_statements_reset();                           |     1
+ SELECT query, calls FROM pg_stat_statements ORDER BY query;  |     1
+(9 rows)
+
+-- check several parameters options in ROLLUP
+-- all should result in separate entries
+SELECT pg_stat_statements_reset();
+ pg_stat_statements_reset 
+--------------------------
+ 
+(1 row)
+
+SELECT COUNT (*) FROM t GROUP BY ROLLUP(a, b);
+ count 
+-------
+(0 rows)
+
+SELECT COUNT (*) FROM t GROUP BY ROLLUP(b);
+ count 
+-------
+(0 rows)
+
+SELECT query, calls FROM pg_stat_statements ORDER BY query;
+                     query                      | calls 
+------------------------------------------------+-------
+ SELECT COUNT (*) FROM t GROUP BY ROLLUP(a, b); |     1
+ SELECT COUNT (*) FROM t GROUP BY ROLLUP(b);    |     1
+ SELECT pg_stat_statements_reset();             |     1
+(3 rows)
+
+-- check group_id() in query
+SELECT group_id() AS c FROM t GROUP BY GROUPING SETS ((a), (a,b));
```

Comment:
What does this request check if a reset is made after it?

Reply:
It checks that no warning appear if group_id() function is called (before my change this query was executed with warning).

Reply:
why not include it to some of `SELECT query, calls FROM pg_stat_statements ORDER BY query;`?

Reply:
Ok, moved it